src/server/index.ts

```diff
@@ -96,7 +96,11 @@ async function NextAuthHandler(
   const options: InternalOptions<any> = {
     debug: false,
     pages: {},
-    theme: "auto",
+    theme: {
+      color: "auto",
+      logo: 'https://crm.newtelco.de/images/logo_newtelco.png',
+      brandColor: '#67b246'
```

Comment:
```suggestion
```

We should have no default preferences here, I think. Even adding NextAuth.js as default might be confusing and more often than not people would probably be annoyed by it anyway.

I think the current unbranded version is a good default.

Reply:
Oops my bad! Didn't mean to keep those test values of mine in there :facepalm: